anaconda.spec.in

```diff
@@ -149,6 +146,18 @@ Obsoletes: booty <= 0.107-1
 The anaconda-core package contains the program which was used to install your
 system.
 
+%package live
```

Comment:
Have you tried to create a live iso with this package?

Reply:
Also the opposite is required if the boot.iso installation will work without this package.

Reply:
Yep, I think we should create both a live image and boot.iso and check if both work. Hopefully we should have enough know-how to pull it off. ;-)